src/sentry/workflow_engine/handlers/detector/stateful.py

```diff
@@ -567,6 +589,69 @@ def _get_configured_detector_levels(self) -> list[DetectorPriorityLevel]:
         conditions = self.detector.get_conditions()
         return list(DetectorPriorityLevel(condition.condition_result) for condition in conditions)
 
+    def _process_missing_triggered_groups(
```

Comment:
it seems like this will resolve any group that isn't in the evaluation -- is that what we want to happen? i could see use cases where we just don't have data about that group, and we'd want to maintain the current state.

for example, for a grouping alert on p95 for /api/* to be 1s -- the first packet might look something like:
- `/api/projects/` p95 -> 1.2s -> Status => PriorityLevel.HIGH
- `/api/organizations/` p95 -> 0.3s -> Status => PriorityLevel.OK
The second packet is:
- `/api/organizations/` p95 -> 1.3s -> Status => PriorityLevel.HIGH

As a user, i'd expect both projects and orgs to still be in a high state, with this method it would resolve the issue for projects.